apps/chat/src/components/Common/ReplaceConfirmationModal/Components.tsx

```diff
@@ -102,6 +105,22 @@ export const EntityRow = ({
     );
   }, [additionalItemData, additionalItemData?.mappedActions, entityId]);
 
+  let qaType = '';
+  switch (type) {
+    case SharingType.Conversation:
+      qaType = 'conversation';
+      break;
+    case SharingType.Prompt:
+      qaType = 'prompt';
+      break;
+    case SharingType.Application:
+      qaType = 'application';
+      break;
+    case SharingType.File:
+      qaType = 'file';
+      break;
+  }
```

Comment:
can't we add data-qa to a <PromptRow />, <ConversationRow />, etc.?

Reply:
thanks, fixed